src/CtaLink/CtaLink.js

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ctaLinkIcons, ctaLinkDisplay, ctaLinkColors, ctaLinkAnimations, ctaLinkButtonSizes, ctaLinkButtonVariants } from './CtaLink.levers';
+import Icon from 'react-hero-icon';
+import clsxd from 'clsx-dedupe';
+
+/**
+ * CTA Link Component
+ *
+ */
+export const CtaLink = ({ classes = {}, text, srText, display, link, color, isButton, variant, size, icon, animate, ref, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const levers = {};
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  // display
+  if (display && ctaLinkDisplay.includes(display)) {
+    switch (display) {
+      case 'inline-block':
+        levers.display = 'su-inline-block';
+        break;
+
+      case 'block':
+        levers.display = 'su-block';
+        break;
+
+      case 'flex':
+        levers.display = 'su-flex';
+        break;
+    }
+  }
```

Comment:
Do you need a display prop if classname is available and only one css class is needed?

Reply:
Yeah probably not needed. I'll likely add a CTA group component later and can control the display through that parent component.

Reply:
This is gone